Libraries/Components/TextInput/TextInput.js

```diff
@@ -861,21 +233,16 @@ const TextInput = createReactClass({
     if (this._rafId != null) {
       cancelAnimationFrame(this._rafId);
     }
-  },
-
-  contextTypes: {
```

Comment:
We still need this in order to have context passed in, right? Interestingly, I don't see any reference to either of these context variables internally and I believe they were part of the initial react sync. 

I'd like to avoid changing logic as much as possible so maybe we should keep these in this PR and remove reference to them in a separate one.